standards/rmrk2.0.0/entities/collection.md

```diff
@@ -35,37 +35,48 @@ A collection MUST adhere to the following standard.
     "type": "string",
     "description": "HTTP(s) or IPFS URI. If IPFS, MUST be in the format of ipfs://ipfs/HASH"
   },
-  "attributes?": {
-    "type": Attribute[],
-    "description": "An array of attribute definitions that get inherited by every NFT in this collection"
+  "properties?": {
+    "type": IProperties,
+    "description": "A map of property object definitions that get inherited by every NFT in this collection"
   }
 }
 ```
 
 ## On-chain Attributes
 
-A Collection can define an array of attributes for NFTs in this collection to inherit. Attributes
-defined this way override attributes in the metadata of individual NFTs, but are overridden further
-by attributes of the same name on the NFT. The order of precedence is, from lowest to highest:
+A Collection can define a map of properties for NFTs in this collection to inherit. Propeties (previously Attributes)
+defined this way override properties in the metadata of individual NFTs, but are overridden further
+by properties of the same name on the NFT. The order of precedence is, from lowest to highest:
 
-- Metadata Attribute
-- Collection-inherited Attribute
-- NFT's own Attribute
+- Metadata Properties
+- Collection-inherited Properties
+- NFT's own Properties
 
 Due to on-chain storage concerns and constraints, this function will usually be reserved only for
 [mutable attributes](../interactions/setattribute.md).

```

Comment:
mutable properties and should we change `setattribute` to `setproperty` or?

Reply:
Yeah I thought about it too, to be honest I hate Property, I liked attribute more. But I wanted to copy what Enjin is doing. maybe this was a mistake, but it's too late now

Reply:
Ok I will change SETATTRIBUTE to SETPROPERTY